src/index.ts

```diff
@@ -1,6 +1,5 @@
 import "dotenv/config"
 import { setupDataDog } from "./config/datadog"
-import { ensureSSL } from "./lib/ensureSSL"
```

Comment:
Is it simply unnecessary, or actually an anti-pattern to use SSL for these within-cluster connections?

Reply:
It's hopefully unnecessary, given how inter-host traffic within the Virtual Private Cloud is already encrypted. But it's not an anti-pattern: the old worries about encryption/decryption performance are no longer a significant concern, according to my understanding. Most importantly: we don't currently _support_ SSL traffic to the internal nginx deployment. That will be addressed by [PLATFORM-2981](https://artsyproduct.atlassian.net/browse/PLATFORM-2981), after which SSL should work internally as well.